BoundingBox bounding_box

float64 longtitude
float64 latitude

int16 image_cols
int16 image_rows

int8 attribute

int8[] image_data